app/src/main/java/com/example/immadisairaj/codeforces/SubmissionActivity.java

```diff
@@ -79,14 +79,14 @@ public void fetchApi() {
             @Override
             public void onResponse(Call<Submission> call, Response<Submission> response) {
 
-                Log.v("url", call.request().url().toString());
+                Log.v(getString(R.string.url), call.request().url().toString());
```

Comment:
these tags can be declared final in the same file instead of resource file

Reply:
Done